examples/twitter-beam-java/src/main/java/ReadFromTweeterDoFn.java

```diff
@@ -0,0 +1,76 @@
+import org.apache.beam.sdk.io.range.OffsetRange;
+import org.apache.beam.sdk.transforms.DoFn;
+import org.apache.beam.sdk.transforms.splittabledofn.RestrictionTracker;
+import org.apache.beam.sdk.transforms.splittabledofn.SplitResult;
+import org.joda.time.Duration;
+import org.joda.time.Instant;
+import twitter4j.Status;
+
+import java.io.IOException;
+import java.io.Serializable;
+import java.util.concurrent.BlockingQueue;
+
+@DoFn.UnboundedPerElement
+final class ReadFromTweeterDoFn extends DoFn<String, String> {
```

Comment:
Perhaps we can add a parameter for a maximum of tweets to read - if it's not set, then read forever - if it's set, then read N tweets. Thoughts?

Perhaps we could also add a parameter to read tweets until a certain date or for a certain duration. Thoughts?